src/main/java/org/broadinstitute/consent/http/resources/DatasetResource.java

```diff
@@ -59,6 +60,35 @@
     private final UserService userService;
     private final DataAccessRequestService darService;
 
+    private String dataSetSampleFileName = "DataSetSample.tsv";
+    private String dataSetSampleContent = "Dataset Name\tData Type\tSpecies\tPhenotype/Indication\t# of participants\tDescription\tdbGAP\tData Depositor\tPrincipal Investigator(PI)\tSample Collection ID\tConsent ID"
+            + "\n(Bucienne Monco) - Muc-1 Kidney Disease\tDNA, whole genome\thuman\tmuc-1, kidney disease\t31\tmuc-1 patients that developed cancer , 5 weeks after treatment\thttp://....\tJohn Doe\tMark Smith\tSC-20658\t1";
+
+    String getDataSetSampleFileName() {
+        return dataSetSampleFileName;
+    }
+
+    void setDataSetSampleFileName(String fileName) {
+        dataSetSampleFileName = fileName;
+    }
+
+    void resetDataSetSampleFileName() {
+        dataSetSampleFileName = "DataSetSample.tsv";
+    }
+
+    String getDataSetSampleContent() {
+        return dataSetSampleContent;
+    }
+
+    void setDataSetSampleContent(String content) {
+        dataSetSampleContent = content;
+    }
+
+    void resetDataSetSampleContent() {
+        dataSetSampleContent = "Dataset Name\tData Type\tSpecies\tPhenotype/Indication\t# of participants\tDescription\tdbGAP\tData Depositor\tPrincipal Investigator(PI)\tSample Collection ID\tConsent ID"
+                + "\n(Bucienne Monco) - Muc-1 Kidney Disease\tDNA, whole genome\thuman\tmuc-1, kidney disease\t31\tmuc-1 patients that developed cancer , 5 weeks after treatment\thttp://....\tJohn Doe\tMark Smith\tSC-20658\t1";
```

Comment:
Instead of copying this twice, make it a private static once. Then, lines 64 and 88 can each point to the same constant.